sdk/eventgrid/azure-eventgrid/azure/eventgrid/_event_mappings.py

```diff
@@ -24,6 +24,12 @@ class SystemEventNames(str, Enum):
     AcsChatThreadCreatedWithUserEventName = (
         "Microsoft.Communication.ChatThreadCreatedWithUser"
     )
+    #backward compatibility
+    AcsChatParticipantAddedToThreadEvent = "Microsoft.Communication.ChatThreadParticipantAdded"
+    AcsChatParticipantRemovedFromThreadEvent = "Microsoft.Communication.ChatThreadParticipantRemoved"
+    # new constants matching the model names
+    AcsChatThreadParticipantAddedEventName = "Microsoft.Communication.ChatThreadParticipantAdded"
```

Comment:
i'm inclined to add these additional constants to match the convention of other consts here - if there's a strong reason not to have two different consts with same value, i'm okay to remove it.